app/helpers/scoreboard_helper.rb

```diff
@@ -43,4 +43,14 @@ def score_cell(record, new:, edit: nil, allowed: false)
       link_to_if allowed, '&mdash;'.html_safe, new
     end
   end
+
+  def format_locking_users(locking_users)
+    locking_users = locking_users.pluck(:name)
+    if current_user.lock_scores?
+      locking_users.delete(current_user.name)
+      "#{locking_users.join(', ')} and you"
+    else
+      locking_users.to_sentence
+    end
+  end
```

Comment:
![image](https://github.com/umts/umasstransit.rodeo/assets/25785789/561745ae-b255-4e0c-bfd3-a1f95ecfce8d)
Take a look at this: https://www.rubydoc.info/docs/rails/Array:to_sentence

Reply:
Yeah I was trying to not have your name be there but seems not smart